mongodb/templates/mongod.conf.j2

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+{# Bind to all ips(default) if in clustered mode,
+   otherwise only to the specified local ip.
+#}
+
+port = {{ mongodb.port }}
+
+# Enable journaling, http://www.mongodb.org/display/DOCS/Journaling
+{% if mongodb.enable_journal %}
+journal = true
+{% else %}
+journal = false
+nojournal = true
+{% endif %}
```

Comment:
Instead of doing an if/else block here, maybe do something like:
```saltstack
journal = {{ mongodb.get('enable_journal', 'false') }}
```
If `journal` is set to false, does `nojournal` still need to be explicitly set to true?